src/NLog/Layouts/LayoutParser.cs

```diff
@@ -512,9 +521,17 @@ private static void MergeLiterals(List<LayoutRenderer> list)
             }
         }
 
-        private static LayoutRenderer ConvertToLiteral(LayoutRenderer renderer)
+        private static LayoutRenderer ConvertToLiteral(IRenderable renderer)
```

Comment:
Think it should stay as type `LayoutRenderer`. Less confusing.

Reply:
not really an opinion on this, so keep it LayoutRenderer